### DISCUSSION TITLE
Device Notes/Comments Field

### DISCUSSION THREAD

**Discussion by smish66** [home-assistant/frontend#9168] [Other feature requests]
Hi

I would like to request if it is possible to maybe add a Comments or Notes field to the Devices page where users could input any specific information about the device that normally wouldn't self populate when the devices is added, i.e. Battery Type, Install Date, Warranty Info, etc. This would be useful for having some of this info quickly/easily on hand rather than having to look it up from separate files or information sheets.

Sorry if this may have already been requested, but I was not able to locate anything similar from a search.

Thanks prior for consideration.

**Comment by spacegaier** [comment]
The tricky part is: Where is that data going to get stored? The HA backend would need to provide a way to do so.
